dbms/src/Server/TCPHandler.cpp

```diff
@@ -731,7 +723,7 @@ void TCPHandler::initBlockInput()
         else
             state.maybe_compressed_in = in;
 
-        state.block_in = std::make_shared<NativeBlockInputStream>(*state.maybe_compressed_in, client_revision);
+        state.block_in = std::make_shared<NativeBlockInputStream>(*state.maybe_compressed_in, 1);
```

Comment:
nit:
Change the codes in `NativeBlockInputStream::readImpl` from `if (server_revision > 0) res.info.read(istr);` to `res.info.read(istr);`. So that we can remove `NativeBlockInputStream::server_revision` and remove the hard code `1` here

Reply:
no, `NativeBlockInputStream` is used by the spill, the revision can not be removed.